app/controllers/admin/administrators/administrators_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Admin
+  module Administrators
+    class AdministratorsController < Admin::BaseController
+      skip_after_action :verify_authorized, only: :index
+      skip_after_action :verify_policy_scoped, except: :index
+
+      def index
+        @administrators = policy_scope(User).admins
+      end
+
+      def new
+        authorize AdminProfile
+
+        @user = User.new(new_user_params)
+        authorize @user
+      end
+
+      def create
+        authorize AdminProfile
+        @user = User.new(create_user_params)
+
+        if @user.invalid?
+          render :new and return
+        end
+
+        redirect_to admin_administrators_confirm_administrator_path(full_name: @user.full_name, email: @user.email)
+      end
+
+    private
+
+      def new_user_params
+        { full_name: params[:full_name], email: params[:email] }
+      end
+
+      def create_user_params
+        params.require(:user).permit(:full_name, :email)
+      end
```

Comment:
Soo, why do we need to have these things separately?

Reply:
One is a submitted form, with a user model. The other is url params to prefill the fields

Reply:
Right, so we use url params to send information from one page we have to another page we have.

I'm not sure this is a great idea (user can bookmark this stuff for example). Could we use session for it? Do we care?

Reply:
Could you describe an issue you think this would cause please?

Reply:
Admin user bookmarks the page using parameters. They finish the flow and create the user.

They later go to this page, and instead of redirecting them to start, it seems to work. It breaks when they try to save the user, due to email constraints.

Reply:
Someone evil sends admin a link with url params for creating an admin. They click the link, go straight to confirmation page. They don't bother reading text in green button. They make admin user that's evil.

Reply:
Will any of these happen? Probably not. Are url params opening us up to both? Yes.

Reply:
Ok, the confirm can be rendered directly from the create. But how would you deal with changing your answers?

Reply:
The way I remember we (well, you) dealt with it in the other flows (adding lead providers, adding delivery partners) we store the answers in session, and we update the session from params when they change. 

Then we clear the session when we complete the flow.

Do you think we could use it here? It feels a bit strong, cause it's just two pages, but I think it would do the trick.